sdk/communication/communication-rooms/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 1.0.0-beta.1 (2022-08-04)
```

Comment:
Make sure to update this date before merging

Reply:
Sure I will keep that in mind